### ISSUE TITLE
vlan tag is not proprogated from peer link to host interface.

### ISSUE THREAD

**Issue by lguohan** [Azure/sonic-sairedis#415]
```
root@5ce48f3510b0:/# tcpdump -evni Ethernet68
tcpdump: listening on Ethernet68, link-type EN10MB (Ethernet), capture size 262144 bytes
03:03:22.011854 02:7c:5e:b5:87:82 > ff:ff:ff:ff:ff:ff, ethertype ARP (0x0806), length 42: Ethernet (len 6), IPv4 (len 4), Request who-has 8.8.8.1 tell 8.8.8.6, length 28
03:03:23.030518 02:7c:5e:b5:87:82 > ff:ff:ff:ff:ff:ff, ethertype ARP (0x0806), length 42: Ethernet (len 6), IPv4 (len 4), Request who-has 8.8.8.1 tell 8.8.8.6, length 28
^C
11 packets captured
11 packets received by filter
0 packets dropped by kernel
root@5ce48f3510b0:/# tcpdump -evni eth19
tcpdump: listening on eth19, link-type EN10MB (Ethernet), capture size 262144 bytes
03:03:36.342362 02:7c:5e:b5:87:82 > ff:ff:ff:ff:ff:ff, ethertype 802.1Q (0x8100), length 46: vlan 8, p 0, ethertype ARP, Ethernet (len 6), IPv4 (len 4), Request who-has 8.8.8.1 tell 8.8.8.6, length 28
03:03:37.366378 02:7c:5e:b5:87:82 > ff:ff:ff:ff:ff:ff, ethertype 802.1Q (0x8100), length 46: vlan 8, p 0, ethertype ARP, Ethernet (len 6), IPv4 (len 4), Request who-has 8.8.8.1 tell 8.8.8.6, length 28
```

**Comment by jipanyang** [comment]
I thought Ethernet68 would be tunneled to eth17?    68/4=17.  then eth17 is peering with namespace sw-srv17 via veth pair.  I should check it again.

**Comment by kcudnik** [comment]
resolved in #417
